app/views/layouts/application.html.erb

```diff
@@ -20,10 +20,10 @@
   <% end %>
   </script>
 
-  <% if Rails.env.development? || Rails.env.staging? %>
+  <% if defined? ENV['BUGMUNCHER_KEY'] %>
```

Comment:
You could drop the `defined?` here. If the key is present this evaluates as truthy.